index.js

```diff
@@ -201,5 +244,10 @@ function isShareCodePattern(code) {
 module.exports = {
 	encrypt: encrypt,
 	decrypt: decrypt,
-	isShareCodePattern: isShareCodePattern
+	isShareCodePattern: isShareCodePattern,
+	_removeHyphens: removeHyphens,
```

Comment:
If they are exported, they are part of the public API and will need to be covered by tests.